src/pages/main/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetingData } from '../../redux/greetingActions';
+
+const Main = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchGreetingData());
+  }, []);
+  const selector = useSelector((state) => state.reducer.greeting);
+  console.log(selector);
```

Comment:
- [ ] <b>OPTIONAL:</b> When you finish implementing the app, it is better to remove all debugging code like `logs` and others. Also, online 10, you would consider using `ShallowEqual` to make sure you re-render your component only when it is necessary(meaning when there are changes in the state).